cmd/server/assets/apikeys/new.html

```diff
@@ -49,6 +49,8 @@ <h1>New API key</h1>
             {{end}}
           </div>
 
+		  {{template "os.html" .}}
```

Comment:
I'm not sure this belongs in the API keys page.

Seems like it should be a separate realm level setting

Reply:
Ok, so it looks like the original proposal I wrote in the issue (to put this in Health Authority) won't work. I'll move to a separate table.

/hold

Reply:
My fault - I didn't get to the proposal on the issue before the PR was sent

Reply:
Moved separately.